Dockerfile

```diff
@@ -0,0 +1,5 @@
+# syntax=docker/dockerfile:1
+FROM python:3.9-alpine
+WORKDIR /app
+COPY . .
+CMD [ "sh", "run_tests.sh" ];
```

Comment:
The main purpose of the Dockerfile is to run the project. So, it should call the main.pay in this case. run_tests.sh is already called in docker-compose.yml. Also there is missing EOF.